lib/rspec/rails/matchers/active_job.rb

```diff
@@ -165,11 +203,51 @@ def queue_adapter
       #       HelloJob.set(wait_until: Date.tomorrow.noon, queue: "low").perform_later(42)
       #     }.to have_enqueued_job.with(42).on_queue("low").at(Date.tomorrow.noon)
       def have_enqueued_job(job = nil)
-        unless ::ActiveJob::QueueAdapters::TestAdapter === ::ActiveJob::Base.queue_adapter
-          raise StandardError, "To use have_enqueued_job matcher set `ActiveJob::Base.queue_adapter = :test`"
-        end
+        check_active_job_adapter
         ActiveJob::HaveEnqueuedJob.new(job)
       end
+      alias_method :enqueue_job, :have_enqueued_job
+
+      # @api public
+      # Passess if `count` of jobs were enqueued
```

Comment:
Typo: Passes

I might rephrase this since `count` is not an argument or anything. How about:

Passes if a job has been enqueued. May chain `at_least`, `at_most` or `exactly` to specify a number of times.